src/inmanta/logging.py

```diff
@@ -340,6 +343,15 @@ def get_bootstrap_logging_config(
         )
         return logging_config_core
 
+    def set_defaults_for_options(self, options: Options) -> Options:
+        """
+        Sets the default value of fields that were not set by the user and have a default value
+        """
+        options.log_file_level = options.log_file_level if options.log_file_level is not None else "INFO"
+        options.timed = options.timed if options.timed is not None else False
+        options.keep_logger_names = options.keep_logger_names if options.keep_logger_names is not None else False
+        return options
+
```

Comment:
This is rather verbose/complex. There is an easier way to determine whether a value is provided on the CLI and that's by relying on the `in` operator. You can for example do `"log_file_level" in options` to know whether the `--log-file-level` option was set.

Reply:
Hum, I was not aware that it worked like that. Your solution is much better, will push the changes now